LogMonitor/LogMonitorTests/ConfigFileParserTests.cpp

```diff
@@ -1523,5 +1523,44 @@ namespace LogMonitorTests
                 Assert::IsTrue(output.find(L"WARNING") != std::wstring::npos);
             }
         }
+
+        ///
+        /// Check that UTF8 encoded config file is opened and read by OpenConfigFile.
+        ///
+        TEST_METHOD(TestUTF8EncodedConfigFileReading)
+        {
+            std::wstring folderName = L"D:\\LogMonitor";

```

Comment:
This will fail on a computer without D drive - look into using %temp%\\LogMonitor instead

Reply:
This has been addressed. 

![image](https://user-images.githubusercontent.com/28774968/185343030-532d524b-f56f-4a34-8acf-84463a0899da.png)